src/app/containers/FrontPageSection/index.jsx

```diff
@@ -72,79 +69,106 @@ const MarginWrapper = ({ firstSection, oneItem, children }) => {
   return children;
 };
 
-const StoryPromoListItem = styled(StoryPromoLi)`
-  ${({ isBulletin, isFirstPromo }) =>
-    isBulletin &&
-    css`
-      @media (min-width: ${GEL_GROUP_1_SCREEN_WIDTH_MIN}) and (max-width: ${GEL_GROUP_2_SCREEN_WIDTH_MAX}) {
-        ${isFirstPromo
-          ? `padding-bottom: ${GEL_SPACING_TRPL};`
-          : `padding: ${GEL_SPACING_DBL} 0 ${GEL_SPACING_TRPL};`}
-      }
-    `}
-`;
-
-const isBulletin = item =>
-  item.contentType === 'TVBulletin' || item.contentType === 'RadioBulletin';
-
-const renderPromo = (item, index, firstSection) => {
-  const topStory = firstSection && index === 0;
-  const lazyLoadImage = !topStory; // don't lazy load image if it is a top story
+const parentGridColumns = {
+  group0: 6,
+  group1: 6,
+  group2: 6,
+  group3: 6,
+  group4: 8,
+  group5: 8,
+};
 
-  if (isBulletin(item)) {
-    return <BulletinContainer item={item} lazyLoadImage={lazyLoadImage} />;
-  }
+const renderPromos = (items, isFirstSection, dir) => {
+  const rows = getRows(items, isFirstSection);
+  const rowsDetails = getRowDetails(rows);
+
+  // Only use StoryPromoUl and Li if there is only one story in one row
+  const sectionHasSingleStory =
+    rowsDetails.length === 1 && rowsDetails[0].stories.length === 1;
+
+  const renderedRows = rowsDetails.map(row => (
+    <row.RowComponent
+      key={row.stories[0].id}
+      stories={row.stories}
+      isFirstSection={isFirstSection}
+      displayImages={row.displayImages}
+      dir={dir}
+      parentColumns={parentGridColumns}
+      parentEnableGelGutters // value is set to true here and passed to each Row component's Grid item
+    />
+  ));
 
   return (
-    <StoryPromoContainer
-      item={item}
-      topStory={topStory}
-      lazyLoadImage={lazyLoadImage}
-    />
+    <MarginWrapper
+      firstSection={isFirstSection}
+      dir={dir}
+      oneItem={!!sectionHasSingleStory}
```

Comment:
From declaration of `sectionHasSingleStory` seems like it can only be a boolean, why do we need to cast it to boolean here again?